### PR TITLE
Final Work for SpaceTravlers

### PR DESCRIPTION
**Pull request by Ashe546**
**What we Implemented**
✅ Created routes and view components
✅ Create directories for all Redux state slice files
✅ Upon the first render fetched data from the SpaceX API endpoints
✅ dispatched an action to store the selected data
✅ Wrote actions and reducers for booking rockets/dragons and joining missions
✅ My Profile section shows the reserved/joined mission, rockets, and dragon
✅ Styled the page based on the layout
✅made tests for all your components and functions

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Ashe546, @Khaled-AlKhateeb, and @tahseenzahra,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no README file. Please implement the [README](https://github.com/microverseinc/readme-template) provided by Microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with a ReadMe that does not conform to the **new** [standardized ReadMe template](https://github.com/microverseinc/readme-template). Kindly ensure to update your ReadMe to this [standardized template](https://github.com/microverseinc/readme-template), and then request another review.

**[New ReadMe Template](https://github.com/microverseinc/readme-template):**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ghiftee — COMMENTED** [review]
Hi there 👋🏽 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not use [Microverse standardized README template](https://github.com/microverseinc/readme-template#readme-top) . I see that you have made some changes based on the previous reviewer's suggestion however, you still need to add the other sections present in the README file. You can find the [standardized README template here](https://github.com/microverseinc/readme-template#readme-top)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/pages/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useSelector } from "react-redux";
```
Comment (Kamba56) [inline src/pages/Profile.jsx:2]: - [ ] Your profile component does not render, it only displays a blank screen. Note that your profile page should display the list of reservations you have made. Kindly fix this, please.
### Your design



### Guideline
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @tahseenzahra, @Ashe546 @Khaled-AlKhateeb, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- Your app complies successfully
- There are no linter errors

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] Your profile page is still blank as the previous reviewer stated. It should display the list of your reserved rockets and missions. Kindly fix it.
Screenshot 2022-11-26 at 08 43 06


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- I suggest that you remove the default react files still present in your app. This would ensure your code is clean.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tahseenzahra** [comment]
> 

Hy @Ghiftee thanks for reviewing it we made the changes as said 
and we made the profile page to display rockets, missions and dragons (as we are 3 partners working on this repo)

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**